data/paymentResponse.go

```diff
@@ -10,20 +10,21 @@ const Penalty = "penalty"
 
 // PaymentResponse represents a response from the payment service GET payment endpoint
 type PaymentResponse struct {
-	Amount                  string       `json:"amount"`
-	AvailablePaymentMethods []string     `json:"available_payment_methods,omitempty"`
-	CompletedAt             time.Time    `json:"completed_at,omitempty"`
-	CreatedAt               time.Time    `json:"created_at,omitempty"`
-	CreatedBy               Created      `json:"created_by"`
-	Description             string       `json:"description"`
-	Links                   PaymentLinks `json:"links"`
-	PaymentMethod           string       `json:"payment_method,omitempty"`
-	Reference               string       `json:"reference,omitempty"`
-	CompanyNumber           string       `json:"company_number,omitempty"`
-	Status                  string       `json:"status"`
-	Costs                   []Cost       `json:"costs"`
-	Etag                    string       `json:"etag"`
-	Kind                    string       `json:"kind"`
+	Amount                  string           `json:"amount"`
+	AvailablePaymentMethods []string         `json:"available_payment_methods,omitempty"`
+	CompletedAt             time.Time        `json:"completed_at,omitempty"`
+	CreatedAt               time.Time        `json:"created_at,omitempty"`
+	CreatedBy               Created          `json:"created_by"`
+	Description             string           `json:"description"`
+	Links                   PaymentLinks     `json:"links"`
+	PaymentMethod           string           `json:"payment_method,omitempty"`
+	Reference               string           `json:"reference,omitempty"`
+	CompanyNumber           string           `json:"company_number,omitempty"`
+	Status                  string           `json:"status"`
+	Costs                   []Cost           `json:"costs"`
+	Etag                    string           `json:"etag"`
+	Kind                    string           `json:"kind"`
+	Refunds                 []RefundResource `json:"refunds"`
```

Comment:
what happens when there are no refunds? Is an empty array or null returned?

Reply:
Empty array. Shall I make it omit empty?

Reply:
Hmm i don't think it does... Just checked it and it returns a null. I actually prefer this due to a potential breaking change. Since it returns null, can we add the omitempty flag?